HelpSource/Classes/Platform.schelp

```diff
@@ -80,6 +80,11 @@ true if the Qt library is available, false otherwise
 method:: hasQtWebEngine
 true if the QtWebEngine library is available, false otherwise
 
+method:: architecture
+A link::Symbol:: naming the architecture for which this version of SuperCollider was built.
+Returns one of 'AArch32' (ARMv7 and earlier), 'AArch64' (ARMv8 and later), 'Itanium64', 'i386', 'x86_64', 'PowerPC',
+or 'unknown' if the architecture is unidentifiable.
+
```

Comment:
I think with the latest code change, `AArch32` now can also refer to ARM v8 in 32-bit mode and `AArch64` refers to ARMv8 in 64-bit mode, right?

Reply:
thanks for catching that! updated to clarify, how does it look now?

Reply:
Looks good!